pages/models.py

```diff
@@ -62,6 +62,16 @@ class BasePage(Page):
 
 class MicrositePage(Page):
     color_help_text = '6 digit CSS color code.'
+    show_accent_border = models.BooleanField(
+        default=False,
+        help_text='Show solid accent border at top of page.'
+    )
+    accent_border_color = button_background_color = models.CharField(
```

Comment:
Suggest to sort fields alphabetically.

Reply:
@cjmabry I just noticed a typo here, can you take a look at this? i don't even understand why this compiles or works at all, seems like invalid syntax?